server/plugin/command.go

```diff
@@ -682,15 +682,15 @@ func getAutocompleteData(config *Configuration) *model.AutocompleteData {
 	subscriptionsAdd.AddTextArgument("Owner/repo to subscribe to", "[owner/repo]", "")
 	subscriptionsAdd.AddNamedTextArgument("features", "Comma-delimited list of one or more of: issues, pulls, pulls_merged, pushes, creates, deletes, issue_creations, issue_comments, pull_reviews, label:\"<labelname>\". Defaults to pulls,issues,creates,deletes", "", `/[^,-\s]+(,[^,-\s]+)*/`, false)
 
-	if config.GitHubOrg != "" {
-		subscriptionsAdd.AddNamedStaticListArgument("exclude-org-member", "Events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)", false, []model.AutocompleteListItem{
+	for _, org := range config.GitHubOrgs {
+		subscriptionsAdd.AddNamedStaticListArgument("exclude-org-member-"+org, "Events triggered by organization members of "+org+" will not be delivered (the organization config should be set, otherwise this flag has no effect)", false, []model.AutocompleteListItem{
```

Comment:
Let's keep that flag as it is for now. The gist of the flag stays the same even with multiple orgs: It limits the subscription to all configured orgs.